### ISSUE TITLE
Add support for math

### ISSUE THREAD

**Issue by hugo-blomdahl** [chrisvel/wreeto_official#10]
Add support for math in notes. There already exists solutions for showing math in webpages ([mathjax.org](https://www.mathjax.org/)). I have also seen math in markdown before, is it something that could be added to this project?

**Comment by chrisvel** [comment]
Of course. I will add it to the roadmap, along with [mermaid.js](https://mermaidjs.github.io/#/) for diagrams.
